SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -257,19 +257,20 @@ void USpatialActorChannel::DeleteEntityIfAuthoritative()
 
 	if (bHasAuthority)
 	{
+		check(Actor != nullptr);
```

Comment:
`Check`s always worry me in repliaction as this will cause a crash if something goes wrong and we're in an editor build. Just verifying this is the behaviour we want versus an error and return?

Reply:
checks don't crash editor builds josh! you can continue past them